[翻译完毕] cpp_style

liaopeiyuan opened the pull request:
Hope this helps! Please pardon my rusty Mandarin....

Some of the translations that I'm unsure of:

destructor: 析构函数
macros： 宏
struct：结构体
ownership：所属关系

liaopeiyuan committed:
[翻译完毕] cpp_style

archibate commented on locales/zh_CN/LC_MESSAGES/cpp_style.po:
```diff
@@ -3,113 +3,117 @@
 
 #: ../../cpp_style.rst:2
 msgid "C++ Style"
 msgstr ""
```
Let's translate the title too:
```suggestion
msgstr "C++ 代码风格规范"
```

archibate commented on locales/zh_CN/LC_MESSAGES/cpp_style.po:
```diff
@@ -3,113 +3,117 @@
+"``new`` & ``free``. Use smart pointers (``std::unique_ptr, std::"
+"shared_ptr`` instead for ownership management)."
 msgstr ""
+"``new`` 和 ``free``。 请使用智能指针（例如``std::unique_ptr, std::"
+"shared_ptr``，而不是手动管理所有关系）"
```
Sorry but not leaving a space between Chinese words and \`\` will cause some *display issues*, please run `make html` and check `build/html/cpp_style.html` to confirm.
```suggestion
"``new`` 和 ``free``。 请使用智能指针（例如 ``std::unique_ptr, std::"
"shared_ptr`` ，而不是手动管理所有关系）"
```

archibate commented on locales/zh_CN/LC_MESSAGES/cpp_style.po:
```diff
@@ -3,113 +3,117 @@
 #: ../../cpp_style.rst:28
 msgid "``NULL``, use ``nullptr`` instead."
-msgstr ""
+msgstr "空指针 ``NULL``。 应当使用``nullptr``。"
```
Same here:
```suggestion
msgstr "空指针 ``NULL``。 应当使用 ``nullptr``。"
```

archibate reviewed (commented):
Thank for translating this! Looks great in general except for a few nits.

archibate commented:
> destructor: 析构函数
> macros： 宏
> struct：结构体
> ownership：所属关系

These are all very good translations, thank you!

archibate committed:
Apply suggestions from code review

Hello? Sorry about applying suggestions on my own, merging this to prevent further conflicts with #43.
